index.md

```diff
@@ -4,14 +4,15 @@ title: Home
 permalink: /
 ---
 
-<section class="cancellation-announcement">
-  <p>Due to the uncertainty surrounding COVID-19 we have decided to reschedule the next RubyConfTH, originally planned for October 2020, to a later date in 2021 to allow us to prepare the best possible conference for you.</p>
-  <p>Bangkok.rb plans to organize a variety of online and, once possible, offline events in Bangkok for the Ruby community <br /> during the rest of 2020.</p>
+<section class="conference-announcement">
+  <p>A comeback of RubyConf TH for 2022 is in the works 🎉</p>
+  <p>The conference is scheduled to take place on December 10-11th 2022 at <a href="https://www.pullmanbangkokkingpower.com/">Pullman Bangkok King Power</a> in Bangkok</p>
+  <p>Standard ticket price is <mark>3,200 THB</mark> for the two-day event inclusive of an international buffet on both days and a welcome party 🥳</p>
```

Comment:
(Approx 89 USD)

inclusive of all talks, an international lunch buffet on both days and a ticket to the official party

Reply:
All changed in [d0069b6](https://github.com/bangkokrb/rubyconfth/pull/174/commits/d0069b693e9dad51b2ce1a117901f360d12fc1c7)